src/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const { Router } = require("express");
```

Comment:
Very picky point, but try to leave a line between internal and external imports. Here you don't have so many so it's fine, but going forwards you're going to be importing a lot of different things, so it helps to keep them separate.